### DISCUSSION TITLE
Hide Declined Events is not working

### DISCUSSION THREAD

**Discussion by Bastian007** [totaldebug/atomic-calendar-revive#1173] [Feature Requests]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

"Show declined events" is inactive but the Dashboard card is showing declined events (comming from Google Calendar).
Only by deletion of the event in source calendar ensures those declined/deleted events are not showing up.
![2023-01-03_14-05-40](https://user-images.githubusercontent.com/12425311/210363765-e3b6c91f-3360-48fb-8e38-5715994b35b8.jpg)
![2023-01-03_14-08-14](https://user-images.githubusercontent.com/12425311/210363786-c7c3d949-77e1-42ba-9cee-b79519b63150.jpg)

First I thought it applies only for recurring events, but all other one-time events are impacted as well.

### Expected behaviour

Hide declined events - inaktive option "Show declined events" treats declined events correctly when Calendar entries are feet by Google Calendar.

### Card Version

7.2.0

### Home Assistant Version

2022.12.9

### Configuration

```shell
type: custom:atomic-calendar-revive
name: Termine
enableModeChange: true
entities:
  - entity: calendar.name1_googlemail_com
    name: name1@googlemail.com
    icon: mdi:face-man
    blocklist: Call
  - entity: calendar.name2_googlemail_com_2
    name: name2@googlemail.com
    icon: mdi:face-woman
firstDayOfWeek: 1
maxDaysToShow: 2
refreshInterval: 60
showDate: false
showDeclined: false
sortByStartTime: true
hideDuplicates: true
showEventIcon: false
showRelativeTime: true
showMultiDay: false
showMultiDayEventParts: false
showProgressBar: true
showMonth: true
showWeekDay: true
showNoEventsForToday: true
showCalendarName: false
showHiddenText: true
showWeekNumber: false
calShowDescription: false
showLastCalendarWeek: false
disableCalEventLink: false
disableCalLocationLink: false
disableCalLink: false
untilText: bis
showLoader: false
dateFormat: L
showCurrentEventLine: false
disableEventLink: false
disableLocationLink: true
showFullDayProgress: false
cardHeight: 80%
showDatePerEvent: false
showEventDate: true
hideFinishedEvents: false
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jkrall** [comment]
FYI — this happened due to a change in HA, caused by the new google calendar support that landed last year.
see:  https://github.com/home-assistant/core/issues/71581

Unfortunately, I don't see a corresponding issue in the architecture repo, suggesting that support for attendees data in the calendar API never went any further:  https://github.com/home-assistant/architecture/issues?q=calendar

Looks like someone will need to pick up this ball and run with it... likely by convincing allenporter that we need these additional attributes, and/or getting them included by proposing a PR.  (I'd love to help with this, but I'm not sure that I have the time...)

**Comment by marksie1988** [comment]
I raised this as a feature request as part of #834 under https://community.home-assistant.io/t/expose-more-options-via-the-calendar-api/509491

**Comment by marksie1988** [comment]
Raised another request here: https://github.com/home-assistant/architecture/discussions/963 upvotes on that discussion may increase the likelihood of getting this added
